src/main/java/uk/gov/hmcts/reform/wataskmonitor/services/ResourceEnum.java

```diff
@@ -17,7 +17,11 @@ public enum ResourceEnum {
     AD_HOC_UPDATE_CASE_CCD_ELASTIC_SEARCH_QUERY(
         "adhoc/update-case/ad-hoc-update-case-ccd-elastic-search-query.json"),
     CAMUNDA_HISTORIC_TASKS_PENDING_TERMINATION(
-        "camunda/camunda-historic-task-pending-termination-query.json");
+        "camunda/camunda-historic-task-pending-termination-query.json"),
+    CAMUNDA_TASKS_UNINITIATED(
+        "camunda/camunda-search-cftTaskState-uninitiated.json"),
```

Comment:
I think the cftTaskState value would be `unconfigured` .  May be better to use that in the naming for consistency?

Reply:
The only difference between them one of them has "createdBefore": "*", one of them has "createdAfter": "*", maybe I may use same query file for both job

Reply:
I'll remove this enum CAMUNDA_TASKS_UNINITIATED and use CAMUNDA_TASKS_CFT_TASK_STATE_UNCONFIGURED